### PR TITLE
showCertificate function implementation

### LINKED ISSUE #37419: Implement function to inspect server's SSL certificate
**Use case**
Check and verify server's SSL certificate

**Describe the solution you'd like**
Some function like showCertificate to populate standard certificate fields...

**Additional context**
Addition following #36583

### PR DESCRIPTION
**Pull request by yakov-olkhovskiy**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
showCertificate() function shows current server's SSL certificate

closes #37419

### CODE REVIEW

**Review by antonio2368 — COMMENTED** [review]
Maybe try adding some small test?

File: src/Functions/FunctionShowCertificate.h [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int SUPPORT_IS_DISABLED;
+}
+
+// showCertificate()
+class FunctionShowCertificate : public IFunction
```
[the remark this answers is not in the document]
Comment (yakov-olkhovskiy) [reply src/Functions/FunctionShowCertificate.h:31]: As I see it's a usual practice for functions - their headers are not intended for inclusion anywhere else than their own cpp - functions are added through linkage.

**Comment by yakov-olkhovskiy** [comment]
@antonio2368 regarding tests - this function requires proper SSL setup to work, and output depends on specific server's SSL certificate. I don't see it's possible with simple test.

File: src/Functions/FunctionShowCertificate.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (antonio2368) [reply src/Functions/FunctionShowCertificate.h:31]: Most of them are templated so I'm not sure if that was the reason, but good to know about the linkage.

**Comment by yakov-olkhovskiy** [comment]
@antonio2368 do you think we can merge it? I need your approving review I think :)

**Review by antonio2368 — APPROVED** [review]

**Pull request merged**